homeassistant/components/octoprint/__init__.py

```diff
@@ -179,9 +188,72 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     hass.config_entries.async_setup_platforms(entry, PLATFORMS)
 
+    async def async_pause_print(call: ServiceCall) -> None:
```

Comment:
Maybe these should be button entities instead?

https://developers.home-assistant.io/docs/core/entity/button

That's our new standard for services that don't require any additional parameters.